tests/moxa1210.py

```diff
@@ -55,3 +66,12 @@ def test_WHEN_DI_input_is_switched_on_THEN_only_that_channel_readback_changes_to
                 continue
 
             self.ca.assert_that_pv_is("CH{:02d}:DI".format(test_channel), "Low")
+
+    @parameterized.expand([
+        ("CH{:02d}:DI:CNT".format(channel), channel) for channel in CHANNELS
+    ])
+    def test_WHEN_di_input_is_triggered_THEN_di_counter_increases(self, channel_pv, channel):
+        self.resetDICounters()
```

Comment:
This function appears to reset all of the counters when testing one channel. Might it speed the running of the tests up by only resetting each channel as it's being tested?

Reply:
I changed the reset function to update only the channel the test applies to.